scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -665,15 +825,28 @@ Function RunTransparentGatewayTest
     }
     Invoke-Expression $testCommand | Out-Host
     $testExitCode = $LastExitCode
-    
     PrintLogs $testStartAt $testExitCode
+
+    # run the various leaf device tests
```

Comment:
Do you think it is better to split below RunLeafDeviceTest, so that you can see each result in VSTS pipeline?

Reply:
By doing this it makes it easier for the end user to run locally. That is my primary reasoning. Also this test matrix will grow quite a bit with additional variables like ECC certs etc.